sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/v2/V2SessionCatalogSuite.scala

```diff
@@ -62,23 +62,13 @@ class V2SessionCatalogSuite
   }
 
   private def newCatalog(): TableCatalog = {
-    val newCatalog = new V2SessionCatalog(spark.sessionState)
+    val newCatalog = new V2SessionCatalog(spark.sessionState.catalog, spark.sessionState.conf)
     newCatalog.initialize("test", CaseInsensitiveStringMap.empty())
     newCatalog
   }
 
   private val testIdent = Identifier.of(Array("db"), "test_table")
 
-  test("Catalogs can load the catalog") {
```

Comment:
v2 session catalog won't be load by name, after we remove the config.

Reply:
With your recent changes, I think the session catalog is loaded by name now, so this test should be restored.